packages/tooling/fast-figma-plugin-msft/src/figma/controller.ts

```diff
@@ -1,8 +1,34 @@
 import { Controller } from "../core/controller";
+import { PluginNodeData } from "../core/node";
 import { PluginUIProps } from "../core/ui";
-import { FigmaPluginNode } from "./node";
+import { canHaveChildren, FigmaPluginNode, isInstanceNode } from "./node";
 
 export class FigmaController extends Controller {
+    private syncInstanceWithMaster(target: InstanceNode): void {
+        const source = target.masterComponent;
+
+        function sync(_source: BaseNode, _target: BaseNode): void {
```

Comment:
You probably don't need a `sync` function - this work can just go in the body of `syncInstanceWithMaster`